airbyte/caches/bigquery.py

```diff
@@ -60,5 +66,8 @@ def get_database_name(self) -> str:
     @overrides
     def get_sql_alchemy_url(self) -> str:
         """Return the SQLAlchemy URL to use."""
-        credentials_path_encoded = urllib.parse.quote(self.credentials_path)
-        return f"bigquery://{self.project_name!s}?credentials_path={credentials_path_encoded}"
+        url: URL = make_url(f"bigquery://{self.project_name!s}")
+        if self.credentials_path:
+            credentials_path_encoded = urllib.parse.quote(self.credentials_path)
+            url = url.update_query_dict({"credentials_path": credentials_path_encoded})
+        return str(url)
```

Comment:
From the logs, it appears that windows file paths (which have backslashes) are being munged - possibly because the path is being encoded twice. I believe `update_query_dict()` performs encoding, so we should be able to remove the prior coding.

```suggestion
        url: URL = make_url(f"bigquery://{self.project_name!s}")
        if self.credentials_path:
            url = url.update_query_dict({"credentials_path": credentials_path})

        return str(url)
```